android/src/main/java/com/swmansion/reanimated/layoutReanimation/ReanimatedNativeHierarchyManager.java

```diff
@@ -190,11 +190,11 @@ public synchronized void updateLayout(
     ViewManager parentViewManager = this.resolveViewManager(parentTag);
     String parentViewManagerName = parentViewManager.getName();
     View container = resolveView(parentTag);
-    if (container != null
-        && parentViewManagerName.equals("RNSScreenContainer")
-        && this.mReaLayoutAnimator != null) {
-      this.mReaLayoutAnimator.applyLayoutUpdate(
-          viewToUpdate, 0, 0, container.getWidth(), container.getHeight());
+    if (container != null 
+        && (parentViewManagerName.equals("RNSScreenContainer") || parentViewManagerName.equals("RNSScreenStack")) 
```

Comment:
I believe we checked that you can change checking parent VM to just checking if the child is `RNSScreen` since we want this to work both for `native-stack` and other navigators: 
```suggestion
        && `viewManagerName.equals("RNSScreen")`
```

Reply:
Yes, you are right. I tested it and everything works 🚀